src/app/_components/CardSpecialist/Organization/ExtendedCard.js

```diff
@@ -1,12 +1,25 @@
 import PropTypes from 'prop-types';
+import { Map } from '@components/Map';
 import { CardOrganization } from '@components/CardSpecialist/Organization/Card';
 import { CardModalWrapper } from '@components/CardSpecialist/CardModalWrapper';
 import { organizationPropType } from '@/app/_components/CardSpecialist/prop-types';
+import { addressesToPoints } from '@/utils/common';
 
 export function CardOrganizationExtended({ organization, className }) {
+  const points = addressesToPoints(organization.addresses);
   return (
     <CardModalWrapper className="mt-10 w-full max-w-[1080px] lg:mt-0 lg:w-[1080px]" key={organization?.id}>
-      <CardOrganization organization={organization} className={className} extended />
+      <div className="flex flex-col gap-3 lg:gap-6">
+        <CardOrganization organization={organization} className={className} extended />
+        {points && points.length > 0 && (
```

Comment:
`points && points.length > 0`  -> `points?.length > 0`?

Reply:
Thanks for comment! Seems better, replaced with just `points?.length.`